bootstrap/007_probes.sql

```diff
@@ -291,8 +291,8 @@ BEGIN
     USING (
         SELECT *
         from (values
-                ('Long Queries', 'start_time < dateadd(minute, -10, current_timestamp())'),
-                ('Big Readers', 'bytes_scanned > 10000000000')
+                ('Long Queries', 'start_time < dateadd(minute, -10, current_timestamp()) AND NOT (query_text ilike \'execute streamlit%\')'),
+                ('Big Readers', 'bytes_scanned > 10000000000 AND NOT (query_text ilike \'execute streamlit%\')')
```

Comment:
Do native apps have a large number of bytes scanned?

Reply:
for now, probably not. I just want to avoid the case similar to "long queries", where someday the app uses large number of scan.

Reply:
Yeah, I just double-checked and it looks like `bytes_scanned` is always zero. I don't think this is super distracting from how this probe works, but wanted to be sensitive to adding too many "this but not that" kinds of probes.

Reply:
makes sense. I will remove this pre-cautious condition for "Big Readers" probe.